server/src/libs/routes/successHandler.js

```diff
@@ -0,0 +1,7 @@
+const successHandler = (message, status, data) => ({
+  Data: data,
+  Message: message,
+  Status: status,
```

Comment:
Is this required?

Reply:
It would help in handling data throughout the project I guess. Should I remove it?

Reply:
Response status is the standard way of doing, we need not put additional status I think

Reply:
Okay, I'll remove it.